tests/integration/onprem/test_pvc_mounting.py

```diff
@@ -74,7 +74,7 @@ def submit_jobs_with_pvc(capsys, cleanup=False, namespace="default", # pylint:di
             0].spec.template.spec.containers[0].volume_mounts[0].mount_path
 
 
-class TestServe(object):
+class MyTestServe(object):
```

Comment:
@abhi-g In fact that's not required, since just feel not good start `Test*` in tests if that's not a test case. Anyway, that's OK to keep this. rollbacked, Thank you.